radis/api/hdf5.py

```diff
@@ -367,7 +367,11 @@ def load(
             raise NotImplementedError(output)
 
         if output == "vaex":
-            df = df.extract()  # return DataFrame containing only the filtered rows
+            # df = df.extract()  # Not required # return DataFrame containing only the filtered rows
+            if columns:  # load only these columns (if they exist)
+                columns = [c for c in columns if c in df.columns]
```

Comment:
@1someshverma just so you don't forget

Reply:
When output and engine are both vaex, fetch_databank() was returning all the columns. So, I added this line. 
Can you specify of which file line ```1329``` I need to check.

Reply:
for vaex engine=output so that line ```329``` is not executed for it .

```
        if engine == output:
            df = df
        elif engine == "vaex":
            # in vaex, column selection has to happen now
            if columns:  # load only these columns (if they exist)
                columns = [c for c in columns if c in df.columns]
            if output == "pandas":
                df_pandas = df.to_pandas_df(column_names=columns)
                df.close()
                df = df_pandas
```

Reply:
Alright. Then I would simply add a `if engine = "vaex"` at the start. This would make the code just easier to read.